doc/devel/add_new_sink.md

```diff
@@ -18,7 +18,7 @@ We will not merge new code in the Cygnus repository coming from a different path
 
 ### Coding style 
 
-Please, add the `fiware-connectors/flume/telefonica_checkstyle.xml` to you IDE as check style configuration. This XML file contains all the coding style rules accepted by Telefónica.
+Please, add the `fiware-cygnus/telefonica_checkstyle.xml` to you IDE as check style configuration. This XML file contains all the coding style rules accepted by Telefónica.
```

Comment:
I have remove some 'flume/' references in the doc/, but probably there are some that I haven't spoted or are situated in other directories.

Similarly, I have changed the repository name in docs for fiware-connectors to fiware-cygnus, anticipating the new name at the end.